### ISSUE TITLE
Model overfitting

### ISSUE THREAD

**Issue by giavudangle** [giavudangle/facemask-detector-mobilenetv2#1]
Val loss > Train loss
Fix splitting data
